contracts/cw1-whitelist/src/error.rs

```diff
@@ -0,0 +1,11 @@
+use cosmwasm_std::StdError;
+use thiserror::Error;
+
+#[derive(Error, Debug)]
+pub enum ContractError {
```

Comment:
If the only mapping is `StdError::Unauthorized` -> `ContractError::Unauthorized` and no custom cases, I would rather just use StdError here. No need to use custom messages if we don't need them. (It would also make inheritance in cw1-subkeys easier... we can use the `cw20-base` -> `cw20-staking` extension as a good example of complex error extension.

Reply:
Problem is AFAIK `StdError::Unauthorized` is triggering a deprecation warning.

Reply:
Ah, good point. We never should create `StdError::Unauthorized`.

I just saw the deprecation warning, and understand that. Good as you have it.